src/Cake.Prca.Tests/FakeCodeAnalysisProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cake.Prca.Tests
+{
+    using System.Collections.Generic;
+    using Core.Diagnostics;
+    using Prca.Issues;
+
+    public class FakeCodeAnalysisProvider : CodeAnalysisProvider
+    {
+        private readonly List<ICodeAnalysisIssue> issues = new List<ICodeAnalysisIssue>();
+
+        public FakeCodeAnalysisProvider(ICakeLog log) 
+            : base(log)
+        {
+        }
+
+        public FakeCodeAnalysisProvider(ICakeLog log, IEnumerable<ICodeAnalysisIssue> issues)
+            : base (log)
```

Comment:
I don't say that we shouldn't apply some rules also to test assemblies, But same ruleset would eg mean that every class and method needs to be fully documented which doesn't make sense.